gamemode/modules/fadmin/fadmin/messaging/sh_shared.lua

```diff
@@ -18,7 +18,9 @@ FAdmin.Messages.MsgTypesByName = {
 function FAdmin.PlayerName(ply)
     if CLIENT and ply == LocalPlayer() then return "you" end
 
-    return isstring(ply) and ply or not IsValid(ply) and "unknown" or ply:EntIndex() == 0 and "Console" or ply:Nick()
+    if isstring(ply) then return ply end
+
+    return isentity(ply) and (ply:EntIndex() == 0 and "Console" or ply:Nick()) or "unknown"
```

Comment:
Hypothetically you could throw a non-player entity in here and have it error on `ply:Nick()`. It would breach the idea of getting the player name, though, so I think it's fine.